deploy/development/deployment.tpl.yml

```diff
@@ -21,6 +21,7 @@ spec:
         app: ${KUBE_NAMESPACE}
     spec:
       terminationGracePeriodSeconds: 35
+      serviceAccountName: cd-serviceaccount
```

Comment:
This is required for the deployment to work - it was actually broken when merged yesterday, but I didn't notice as we can't see the deployment failing (in some ways this is good, it shows us that a broken deployment won't go through, and the existing version will remail)